### PR TITLE
The return of Snow Bullet SFX

### PR DESCRIPTION
**Pull request by strubium**
## 🤔 What type of PR is this? (check all applicable)

- [x] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🐛 Bug Fix
- [ ] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [x] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

This adds a (Fair use) sound effect for Snow and Crafted Snow.

## 🧐 The Rationale

MWC reuses lots of sound effects, which leads to "sound fatigue" of the players. This adds a new sound effect to spice it up a little. 

## 🎯 Key Objectives

- Add bullet_14_snow
- Add MaterialImpactSound for Crafted Snow

## 🚦 Testing

I have not tested this yet! 

## ⏮️ Backwards Compatibility 

1.12.2

## 📚 Related Issues & Documents

N/A

## 🖼️ Screenshots/Recordings

Will provide when i test my changes

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

## Summary by CodeRabbit

- **New Features**
	- Introduced a new sound effect for interactions with Snow in the game.

### CODE REVIEW

**Comment by Desoroxxx** [comment]
Any progress on this, getting this into 0.1 would be very nice

**Comment by strubium** [comment]
> Any progress on this, getting this into 0.1 would be very nice

Tested it, it works.

**Review by Paneedah — APPROVED** [review]
This looks good! Don't have any complaints.

**Comment by Desoroxxx** [comment]
Emotional Damage

**Comment by Desoroxxx** [comment]
Anyway you can fix this?

**Comment by Koud-Wind** [comment]
Remove the code from line 100 of src/main/java/com/paneedah/mwc/MWC.java

**Review by SticksDeveloper — APPROVED** [review]
Looks good to me

File: src/main/java/com/paneedah/mwc/MWC.java [hunk 1]
```diff
@@ -99,6 +99,8 @@ public void init(FMLInitializationEvent initializationEvent) {
         modContext.setMaterialImpactSounds(Material.SAND, 1.5f, "bullet_5_grass", "bullet_9_grass", "bullet_11_grass", "bullet_10_snap", "bullet_13_snap");
         modContext.setMaterialImpactSounds(Material.SNOW, 1.5f, "bullet_5_grass", "bullet_9_grass", "bullet_11_grass", "bullet_10_snap", "bullet_13_snap");
```
Comment (Koud-Wind) [inline src/main/java/com/paneedah/mwc/MWC.java:100]: Remember to delete it

Comment (strubium) [reply src/main/java/com/paneedah/mwc/MWC.java]: thank you lmao

**Review by Desoroxxx — APPROVED** [review]

**Pull request merged**